cabal-install/Distribution/Client/DistDirLayout.hs

```diff
@@ -107,14 +111,20 @@ data DistDirLayout = DistDirLayout {
      }
 
 
+data StoreDirLayout = StoreDirLayout {
```

Comment:
Well, apparently having really long rpaths can slow start times down a bit, so it would be nice if we could get the best of both worlds. :/